kolibri/core/device/api.py

```diff
@@ -193,34 +197,67 @@ class Meta:
         fields = ["user", "member_of"]
 
 
+RECENTLY_SYNCED = "RECENTLY_SYNCED"
+SYNCING = "SYNCING"
+QUEUED = "QUEUED"
+NOT_RECENTLY_SYNCED = "NOT_RECENTLY_SYNCED"
+
+
+def map_status(status):
+    """
+    Summarize the current state of the sync into a constant for use by
+    the frontend.
+    """
+    if status["active"]:
+        return SYNCING
+    elif status["queued"]:
+        return QUEUED
+    elif status["last_synced"]:
+        if timezone.now() - status["last_synced"] < timedelta(hours=1):
```

Comment:
I'm thinking it might be good to have the time delta be a configurable value, since "recent" is contextually variable.

Reply:
@jamalex - that's a great point and a good catch, because this is actually an arbitrary interval that I made up while I was coding. I had asked Khang if he had an interval in mind of what "not recently synced" meant, and as I recall, he didn't think there had been a decision about that, and then I completely forgot to follow up with folks... 🙃 So, perhaps there could be some discussion/options about that now.

Reply:
I had wondered about the somewhat arbitrary '60 days' in milliseconds encoding in the frontend wrt to this. I assumed that 60 days was probably not what we wanted, but the exact time does seem to be context dependent.